### DISCUSSION TITLE
How to control the camera in react?

### DISCUSSION THREAD

**Discussion by reginna-chao** [splinetool/react-spline#83] [Q&A]
If the spline file already set special camera, How to control the time timeStamp or keyframes? The camera is move too slow. (I can't edit spline project.)
![image](https://user-images.githubusercontent.com/19789757/192169643-de23f63b-b08b-407b-b8de-3d3815509f8a.png)

**Comment by marcofugaro** [comment]
You should be able to edit the camera animation inside the Spline editor.
